### DISCUSSION TITLE
readMESH gets caught in endless loop

### DISCUSSION THREAD

**Discussion by dyollb** [libigl/libigl#2141] [Q&A] [answered]
hi

I am trying to run the direct delta mush tutorial for my own model.
In order to get the weights I meshed the model using tetgen and am running the biharmonic weights tutorial for my mesh (and tgf file).

The mesh is not very big:
> Mesh points: 23616
  Mesh tetrahedra: 81744
  
But maybe too big for Bounded Biharmonic Weights?

What a reasonable mesh size for this? Can I enable a different solver to get higher performance?
My specs:
- MacOS 12.6.2, clang-1400.0.29.202
- 16GB RAM, 2.8 GHz Quad-Core

**Comment by alecjacobson** [comment]
Could you share some example input and minimal code snippet ?

**Comment by dyollb** [comment]
I am still investigating, but it seems the readMESH function is caught in an endless loop (while ignoring comments/empty lines). i never eeached the bbw function.

The mesh was ceeated with tetgen (libigl wrapper) and saved using writeMESH.

**Comment by dyollb** [comment, marked as answer]
Fixed by #2079
